.github/workflows/fetch_versions.yml

```diff
@@ -22,6 +22,20 @@ jobs:
         uses: actions/checkout@v3
         with:
           token: ${{ steps.generate_token.outputs.token }}
+      - name: Install Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19.x
+          check-latest: true
+      - name: Cache
+        uses: actions/cache@v3
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-plugins-fetch-versions-${{ hashFiles('**/go.sum') }}
+          restore-keys: ${{ runner.os }}-plugins-ci
+      - name: Install buf cli
+        run: |
+          go install github.com/bufbuild/buf/cmd/buf@main
```

Comment:
Do we need main? Or should we use https://github.com/bufbuild/buf-setup-action

Reply:
We need main because it has support for some newer fields in `buf.plugin.yaml` (like licenses, etc.) that haven't made it into a release yet.